mteb/models/promptriever_models.py

```diff
@@ -19,18 +19,19 @@ class PromptrieverWrapper(RepLLaMAWrapper, Wrapper):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-    def encode_queries(self, queries: list[str], **kwargs: Any) -> np.ndarray:
-        queries = [f"query:  {query}" for query in queries]
-        if "instruction" in kwargs:
-            end_punct_list = [
-                "?" if query.strip()[-1] not in ["?", ".", "!"] else ""
-                for query in queries
-            ]
-            queries = [
-                f"{query}{end_punct_list[i]} {kwargs['instruction']}"
-                for i, query in enumerate(queries)
-            ]
-        return self.encode(queries, **kwargs)
+    def encode(
+        self,
+        sentences: list[str],
+        *,
+        task_name: str,
+        prompt_type: PromptType | None = None,
+        **kwargs: Any,  # noqa
```

Comment:
```suggestion
        **kwargs: Any,
```

Reply:
What is the problem here?

Reply:
I copied this from the RepLLaMA, at this point I am not sure. Will check and remove if not needed.